src/Internal/Cardano/Types/Transaction.purs

```diff
@@ -106,13 +118,25 @@ import Ctl.Internal.Serialization.Address
   , StakeCredential
   )
 import Ctl.Internal.Serialization.Hash (Ed25519KeyHash)
-import Ctl.Internal.Serialization.Types (VRFKeyHash)
+import Ctl.Internal.Serialization.Keys
+  ( bech32FromEd25519Signature
+  , bech32FromPublicKey
+  , bytesFromPublicKey
+  )
+import Ctl.Internal.Serialization.ToBytes (toBytes)
+import Ctl.Internal.Serialization.Types
+  ( Ed25519Signature
+  , PublicKey
+  , VRFKeyHash
+  ) as S
```

Comment:
I'd personally prefer to avoid one-letter module aliases as they can be quite ambiguous (unless it's a widespread convention, like `T` for `Data.Text` in Haskell). I know that there are several examples of that throughout the codebase, but moving forward I'd like to avoid adding anymore